### PR TITLE
update: ref -> reference

### PR DESCRIPTION
**Pull request by t-h2o**
# update: ref -> reference

Was it forgotten?

According to #1819

### CODE REVIEW

File: doc/CHANGELOG.md
Comment (mwestphal) [inline doc/CHANGELOG.md:1]: This one should not be changed, it's the changelog

File: doc/dev/TESTING.md
Comment (mwestphal) [inline doc/dev/TESTING.md:1]: This one indeed was forgotten

**Review by mwestphal — CHANGES_REQUESTED** [review]
small changes needed

File: doc/dev/TESTING.md
Comment (t-h2o) [reply doc/dev/TESTING.md:1]: I keep the change

File: doc/CHANGELOG.md
Comment (t-h2o) [reply doc/CHANGELOG.md:1]: I reset as before in a new commit

**Comment by t-h2o** [comment]
Good, changes are done

**Review by mwestphal — APPROVED** [review]

**Comment by mwestphal** [comment]
Thanks for your contribution @t-h2o :)

**Pull request merged**